crates/autopilot/src/driver_api.rs

```diff
@@ -3,17 +3,29 @@ use {
     anyhow::{anyhow, Context, Result},
     reqwest::Client,
     shared::http_client::response_body_with_size_limit,
+    std::time::Duration,
     url::Url,
 };
 
 const RESPONSE_SIZE_LIMIT: usize = 10_000_000;
+const RESPONSE_TIME_LIMIT: Duration = Duration::from_secs(60);
```

Comment:
I think we should probably have different timeouts for different calls. (`/quote`, `/solve`, `/settle`)

Reply:
In general, calls to the endpoints should probably enforce their own timeouts through tokio::timeout. This constant was meant as a safety fallback for all routes.